test/cypress/integration/widgets/blueprints_spec.ts

```diff
@@ -420,7 +419,36 @@ describe('Blueprints widget', () => {
         });
     });
 
+    describe('should open Composer', () => {
+        before(() => {
+            cy.reload();
+        });
+
+        it('after click menu item', () => {
```

Comment:
To keep test naming consistent (with e.g. "should open Composer with the blueprint imported on "Edit a copy in Composer" icon click"), I'd suggest to change it to:
```suggestion
        it('on "Generate in the Composer" menu item click', () => {
```